patch/files/scripts/jobs&specs.gd

```diff
@@ -563,6 +563,9 @@ var weakitemslist2 = ['aphrodisiac','hairdye', 'hairgrowthpot', 'stimulantpot',
 var gearitemslist = ['clothsundress','clothmaid','clothkimono','clothmiko','clothbutler','underwearlacy','underwearboxers','armorleather','armorchain','weapondagger','weaponsword','accslavecollar','acchandcuffs']
 var ingredlist = ['bestialessenceing', 'natureessenceing','taintedessenceing','magicessenceing','fluidsubstanceing']
 
+var sexactstoday = 0 #ralphC - added for Succubus meal count
```

Comment:
This variable never requires the use of this scope, thus is should be declared in each function that it is used.

Reply:
I thought I was saving lines of code by declaring it one level higher.  Preference noted and comment incorporated for next commit.